server/repository/assessment/removeAssessmentCycle.ts

```diff
@@ -0,0 +1,13 @@
+import { Cycle } from '@core/meta/assessment/cycle'
+import { BaseProtocol, DB } from '@server/db'
+
+export const removeAssessmentCycle = async (
```

Comment:
this method must return read from './read' which will return the assessment with the cycles